.github/workflows/release.yml

```diff
@@ -0,0 +1,55 @@
+---
+name: Release
+
+on:
+  pull_request:
+    types:
+      - closed
+    branches:
+      - master
+      - "[2-9][0-9].[0-9][0-9].x"
+    paths:
+      - "centreon-awie/**"
+      - "centreon-dsm/**"
+      - "centreon-gorgone/**"
+      - "centreon-ha/**"
+      - "centreon-open-tickets/**"
+      - "centreon/**"
+  workflow_dispatch:
+
+jobs:
+  release:
+    if: ${{ github.event.pull_request.merged == true }}
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Check base_ref
+        run: |
+          set -eu
+
+          # Check if github.base_ref is either master or any of the supported version ones
+          # This must never run on any other than master and supported version base_ref
+          if [[ ${{ github.base_ref }} == 'master' || ${{ github.base_ref }} =~ '[2-9][0-9].[0-9][0-9].x' ]];then
```

Comment:
```suggestion
          if [[ ${{ github.base_ref }} == 'master' || ${{ github.base_ref }} =~ ^[2-9][0-9].[0-9][0-9].x ]];then
```
Did you try this ?

Reply:
this was moved to a dedicated step (as no option exist to handle regex patterns in job conditionals directly)